src/Benchmark/StatSendingBenchmark.cs

```diff
@@ -2,13 +2,16 @@
 using BenchmarkDotNet.Attributes;
 using JustEat.StatsD;
 using JustEat.StatsD.EndpointLookups;
+using JustEat.StatsD.V2;
```

Comment:
Perhaps the new publisher, transport types and namespaces should have a name other than V2, naming stuff is hard though.

Something that indicates it offers better perf / zero allocations.

Reply:
+1, V2 is confusing given that the library is at version 3.